docs/results.md

```diff
@@ -0,0 +1,376 @@
+        + [JSON data](#json-data)
+  * [puppet-metrics-collector](#puppet-metrics-collector)
+  * [Tune settings](#puppet-metrics-collector)
+- [Scale](#scale)
+  * [Scale test report](#scale-test-report)
+  * [Beaker environment settings](#beaker-environment-settings)
+  * [Iterations](#iterations)
+    + [Iteration sub-directories](#iteration-sub-directories)
+      * [Host directories](#host-directories-1)
+        + [Master](#master-1)
+        + [Metric](#metric-1)
+  * [Gatling JSON data](#gatling-json-data)
+  * [pe_tune](#pe_tune)
+  * [puppet-metrics-collector](#puppet-metrics-collector-1)
+
+
+# Background
+[GPLT](https://github.com/puppetlabs/gatling-puppet-load-test) has two high-level test types: [Performance](#performance) and [Scale](#scale). 
+The test results are organized into separate directories for these test types.
+
+This document currently covers the Standard reference architecture. 
+[SLV-698](https://tickets.puppetlabs.com/browse/SLV-698) has been created to update it for the Large reference architecture.
+
+# Performance
+Performance tests consist of a single iteration of a scenario such as the [Apples2Apples](https://github.com/puppetlabs/gatling-puppet-load-test#apples-to-apples-performance-tests) and [Soak](https://github.com/puppetlabs/gatling-puppet-load-test#soak-performance-tests) tests. 
+The results for Performance test runs can be found in the [`results/perf`](https://github.com/puppetlabs/gatling-puppet-load-test/tree/master/results/perf) directory.
+
+The results for each Performance test run are contained within a separate timestamped sub-directory of the `results/perf` directory.
+The name of each result directory consists of the label `PERF_` to indicate a Performance run followed by a timestamp generated at the start of the run.
+For example:
+```
+results/perf
+└── PERF_1572446968
+```
+
+Performance result directories contain multiple files and sub-directories, each of which will be explained below.
+For example:
+  
+```
+results/perf/PERF_1572446968
+├── 20191030T144928Z-20191030T184941Z.tar.gz
+├── current_tune_settings.json
+├── end_epoch
+├── ip-10-227-0-214.amz-dev.puppet.net
+├── ip-10-227-0-248.amz-dev.puppet.net
+├── puppet-metrics-collector
+├── puppetserver.average.csv
+├── puppetserver.average.csv.html
+├── puppetserver.csv
+├── puppetserver.csv.html
+└── start_epoch
+```
+
+## Host directories
+The results directory contains sub-directories for each PE infrastructure host.
+In the Standard reference architecture example above this would be the PE 'master'.
+A sub-directory for the 'metric' node (where the Gatling simulation is run) is also included.
+The name of each sub-directory corresponds to the hostname of the node.
+
+### Master
+#### atop measurements
+The 'master' node directory contains the atop measurements provided by [beaker-benchmark](https://github.com/puppetlabs/beaker-benchmark) and additional CSV and HTML files generated from these files.
+For example:
```

Comment:
I think you want a blank line before this so that it doesn't get folded into the previous paragraph when rendered.